tests/test_google_analytics_dynamic_dimension_metric.py

```diff
@@ -0,0 +1,87 @@
+import tap_tester.connections as connections
```

Comment:
I think this test is a better implementation of the all_fields test for this tap. I don't think both are needed. This one should remain but the all fields test should be removed.

Reply:
@kspeer825 As per your comment removed all fields test cases.